cfpb/cfgov-refresh
# Fix for failing Youtube API

The Youtube API sometimes fails to load which causes problems when accessing its video player API. This is a workaround for that issue which removes and reattaches the iframe when stopping / playing.
I also changed hardcoded `http` references to `https`. 

## Changes

-  Modified `cfgov/unprocessed/js/modules/UStreamPlayer.js` to switch `http` to `https`.
-  Modifeid c`fgov/unprocessed/js/modules/YoutubePlayer.js` to attach / reattach the iframe when stopping / playing.

## Testing

- Modify the `origin` property in `video-player.html`
- Visit `http://localhost:8000/about-us/the-bureau/`.
- You should be able to play and stop the video without error.
- Check the console to verify that there are no errors.


## Checklist
